### PR TITLE
Add_styling_for_desktop_using_CSS_media_query_CSS_styling_and_HTML

### PR DESCRIPTION
**Pull request by Munsa1**
Hello😍
In the following pull request, I have done the following

- Used CSS media query to make my project look like the template I selected
- Used a grid CSS system to display my cards to look like the cards in the template I chose.
- Used the fonts used in the template I selected.
- Used flexbox to layout my HTML elements as much as possible
- Done a linter error check locally to ensure clean code
- Tried to stick to the original template design as much as possible

Please have a look your feedback is highly appreciated.

Good Day👏

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -147,8 +176,9 @@ <h5 class="card-heading heading-color">Skills</h5>
 </main>
 
 <footer class="footing background">
+    <div class="footer-main-container">
 <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-<div>
+<div class="footer-right-container">
 <form class="form-container user-form" action="https://formspree.io/f/xbjqyrpj" method="POST">  
     <input type="text" class="user-info name" placeholder="Full name" name="user-name" maxlength="30" required>
```
Comment (mricanho) [inline index.html:183]: - [ ] For the desktop version you need to add an extra input, one for your first name and the other for your last name

**Your design:**

image

**Original design:**

image

File: index.html [hunk 2]
```diff
@@ -147,8 +176,9 @@ <h5 class="card-heading heading-color">Skills</h5>
...
     <input type="email" class="user-info" placeholder="Email address" name="email-address" required>
```
Comment (mricanho) [inline index.html:184]: - [ ] Move the "get in touch" button from the center to the left

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Munsa1 @ntzwilly, 

Overall, you did a very good job with the desktop version, let me congratulate you! 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

- [ ] **[OPTIONAL] You can set a `button { cursor: pointer}` for the desktop version, it's for UX reasons**

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

surprised


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Munsa1) [reply index.html:184]: I have done it, thank you

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
Hello @Munsa1 

##### Great work on making the changes requested by the previous reviewer:clap::clap::clap:
You've done a good job implementing some of the requested changes:blush:
but there are still some which aren't done yet, 
so we need to implement those changes to get the project approved:muscle::muscle::muscle:

 ## Required Changes ♻️

- [ ] You still need to align the get in touch button a little bit more to the left.
- [ ] The form should have the same width as the above section.

#### Original Design
right

#### Your project
wrong


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Review by Shaher-11 — APPROVED** [review]
Hello @Munsa1 ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**